content/lessons/chapter-3/pool-1.tsx

```diff
@@ -110,24 +191,74 @@ export default function POL1({ lang }) {
               />
             </Card>
             <Card className="flex">
-              <span className="fade-in font-nunito text-[15px] font-bold text-white text-opacity-25">
+              <span
+                className={clsx('fade-in font-nunito text-[15px] font-bold', {
+                  'text-white text-opacity-25': step === 0,
+                  'fade-in text-[#EDA081]': step !== 0,
+                })}
+              >
                 Hashes
               </span>
               <span className="fade-in font-nunito text-[15px] font-bold text-white text-opacity-25">
-                0
+                <div
+                  className={clsx('font-space-mono font-normal', {
+                    'text-white/25': step === 0,
+                    'fade-in text-white': step !== 0,
+                  })}
+                >
+                  {step === 0
+                    ? 0
+                    : (
+                        profile.hashpower /
+                        10 ** (profile.hashpower.toString().length - 2)
+                      ).toFixed(2)}
```

Comment:
Not sure what is happening here, why are we converting hashpower to a string, then taking the length of the string and subtracing 2. Whenever we're doing magic like this (which could very well be necessary) its good to leave some comments so others know what the magic is doing :smile: